config/kustomization/namespaced/controller-rbac.yaml

```diff
@@ -0,0 +1,68 @@
+# Copyright 2023 TriggerMesh Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    app.kubernetes.io/part-of: triggermesh
+  name: triggermesh-controller-watch
+rules:
+# List and watch receive-adapters
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  verbs:
+  - list
+  - watch
+  # knative core objects
+- apiGroups:
+  - serving.knative.dev
+  - eventing.knative.dev
+  - messaging.knative.dev
+  - messaging.triggermesh.dev
```

Comment:
Do we have this group or domain? Couldn't find it anywhere

Reply:
them come from the `addressable-resolver` kn clusterrole. This is the binding https://github.com/triggermesh/triggermesh/blob/main/config/202-clusterrolebindings.yaml#L101-L115